database/dbase.go

```diff
@@ -344,7 +344,7 @@ func (rcdb *ReconnectableCloudtrustDB) Ping() error {
 	}
 	err = dbConn.Ping()
 	if err != nil {
-		rcdb.resetConnection(true)
+		_ = rcdb.resetConnection(true)
```

Comment:
?

Reply:
Reverted to keep sonar code smell and process it correctly when we'll have time